docs/resource/os.md

```diff
@@ -15,6 +15,7 @@ In case of virtualized environments, this is the operating system as it is obser
 | `os.description` | string | Human readable (not intended to be parsed) OS version information, like e.g. reported by `ver` or `lsb_release -a` commands. | `Microsoft Windows [Version 10.0.18363.778]`; `Ubuntu 18.04.1 LTS` | Recommended |
 | `os.name` | string | Human readable operating system name. | `iOS`; `Android`; `Ubuntu` | Recommended |
 | `os.version` | string | The version string of the operating system as defined in [Version Attributes](/docs/resource/README.md#version-attributes). | `14.2.1`; `18.04.1` | Recommended |
+| `os.build` | string | Unique identifier for a particular build or compilation of the operating system. | `TQ3C.230805.001.B2`; `20E247` | Recommended |
```

Comment:
nit: Could we add examples/guidance for some common OS? e.g. for macOS, Windows and something like Ubuntu?

Reply:
Thanks @mx-psi for suggesting this. I looked up what syntax [windows OS build numbers](https://en.wikipedia.org/wiki/List_of_Microsoft_Windows_versions) are and what syntax [macOS build numbers](https://apple.fandom.com/wiki/Build_number) are. Looks like they're not very different from the second example. But I went ahead and added a completely number string too (as in not alphanumeric) like in case of windows.